tests/test_lint_recipe.py

```diff
@@ -674,6 +674,57 @@ def test_jinja_load_file_regex(self):
                 )
             lints = linter.main(recipe_dir)
 
+    def test_jinja_load_file_data(self):
+        # Test that we can use load_file_data in a recipe. We don't care about
+        # the results here and/or the actual file data because the recipe linter
+        # renders conda-build functions to just function stubs to pass the linting.
+        # TODO: add *args and **kwargs for functions used to parse the file.
```

Comment:
This still needs to be done? Where?

Reply:
I'm not sure if I'm understanding your question. 
 
The recipe_lint process fail with an undefined variable/function error, when the recipe is rendered. The render process is just the basic JINJA2 templating elements and it doesn't include the functions defined by the conda-build: [*loading data from other files*](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#loading-data-from-other-files) 

But, the conda-build steps on conda-forge are OK with them: 

An example can be found at:
conda-forge/r-climate4r-feedstock#1

which just still an ongoing PR.

Reply:
You have a TODO in your comment and I don't understand what needs to be done. Can you please explain that?

Reply:
OK. It was just to mention to add linting to the arguments of the functions as a future improvement. But, from your comment, it's a bit confusing and it appears to be something to be done in this PR or just next release. 

Please, feel free to review and remove the TODO.